### PR TITLE
Add info on how to rebase branch

### PR DESCRIPTION
**Pull request by ssh24**
Some contributors have difficulties rebasing their branch for a PR. We should specify these breakdowns on `CONTRIBUTING.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ Contributing to `loopback-datasource-juggler` is easy. In a few simple steps:
 
   * Submit a pull request through Github.
 
+### How to rebase your branch ###
+
+Our code base is always getting updated and being able to successfully rebase your PR is very important. Here are some simple steps to rebase your code:
+- Checkout your branch which you are submitting the pull request against `git checkout <my_branch>`
+- Add an upstream remote from your fork `git remote add upstream https://github.com/strongloop/loopback-datasource-juggler.git`
+- Fetch the latest from the **_upstream_** remote `git fetch upstream`
+- Rebase your code using the interactive mode `git rebase -i uptream/master`
```
Comment (kjdelisle) [inline CONTRIBUTING.md]: Typo on `upstream`

Comment (ssh24) [inline CONTRIBUTING.md]: :/ thank you. fixed it.

**Comment by 0candy** [comment]
Should this be in the docs instead?
https://loopback.io/doc/en/contrib/code-contrib.html

**Comment by dhmlau** [comment]
The content added in this PR is good, I think we should have this for all our repos.  

Having said that, there seems to be duplicating content in the `CONTRIBUTING.md` in our repos.  Also, there is the `Contributing code` in the LB doc. http://loopback.io/doc/en/contrib/code-contrib.html#how-to-contribute-to-the-code

There's nothing blocking this PR to land.  But my suggestion would be: 
- review all the CONTRIBUTING.md in our repos, and see if we can extract the duplicate info into a common place.  
- remove the CLA agreement content because we already have the link to the agreement.  Otherwise, we'd need to change the CONTRIBUTING.md if there is any update to the agreement.

**Comment by ssh24** [comment]
Closing this. Moved the info up to the `loopback.io` level: https://github.com/strongloop/loopback.io/pull/368

**Pull request closed without merging**